WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductFormBottomSheetActionsFactory.swift

```diff
@@ -0,0 +1,17 @@
+import Yosemite
+
+struct ProductFormBottomSheetActionsFactory {
```

Comment:
It would be nice if we have some documentation about what this factory is for. 🙂

Reply:
yup! added in https://github.com/woocommerce/woocommerce-ios/pull/2303/commits/5a0458115cef7247afe72814974d4e2a8db483fa